extensions/brew/src/upgrade.tsx

```diff
@@ -1,15 +1,9 @@
-import { showToast, Toast } from "@raycast/api";
-import { brewUpgradeAll } from "./brew";
-import { preferences } from "./preferences";
-import { showActionToast, showFailureToast, wait } from "./utils";
+/**
+ * Upgrade command entry point.
+ *
+ * This is a view-mode command that displays upgrade progress.
```

Comment:
I think the comment is incorrect: it's a no-view command.

Reply:
That's true for most of the other brew commands though - it would be nice if the API supported longer-running tasks which could continue independently of the view once triggered.